packages/iTwinUI-react/src/core/TimePicker/TimePicker.tsx

```diff
@@ -234,14 +322,74 @@ export const TimePicker = (props: TimePickerProps): JSX.Element => {
     step: number,
   ) => {
     const data = [];
-    for (let i = 0; i < size; ++i) {
+    for (let i = 0; i < size; i++) {
```

Comment:
Why switched?

Reply:
I looked it up and it doesn't matter which way in the context of a for loop, so I just aligned it with the method I wanted to use in other places in this file.